homeWork_03/ex3_Circle/Sergei_Voitenkov/src/Circle.java

```diff
@@ -0,0 +1,9 @@
+public class Circle {
+    public double radius;
+
+    double calculateArea(){
+        double p = 3.1415;
```

Comment:
Our static analysis engine detects that your pull request includes a change that uses the rough value of Math.PI. According to the [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value), it's recommended to use the predefined library constant for code clarity and better precision.